src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -681,13 +679,11 @@ static void Emscripten_UpdatePointerFromEvent(SDL_WindowData *window_data, const
         SDL_SendPenMotion(0, pen, window_data->window, mx, my);
 
         if (event->button == 0) {  // pen touch
-            SDL_SendPenTouch(0, pen, window_data->window, (event->buttons & 1) ? SDL_PRESSED : SDL_RELEASED, 0);
+            SDL_SendPenTouch(0, pen, window_data->window, false, down);
         } else if (event->button == 5) {  // eraser touch...? Not sure if this is right...
-            SDL_SendPenTouch(0, pen, window_data->window, (event->buttons & 32) ? SDL_PRESSED : SDL_RELEASED, 1);
-        } else if (event->button == 1) {
-            SDL_SendPenButton(0, pen, window_data->window, (event->buttons & 4) ? SDL_PRESSED : SDL_RELEASED, 2);
-        } else if (event->button == 2) {
-            SDL_SendPenButton(0, pen, window_data->window, (event->buttons & 2) ? SDL_PRESSED : SDL_RELEASED, 1);
+            SDL_SendPenTouch(0, pen, window_data->window, true, down);
+        } else {
+            SDL_SendPenButton(0, pen, window_data->window, event->button, down);
```

Comment:
@icculus, are these changes right? The masks and button ids in the original code seemed wrong, but...?

Reply:
Nope, it needs all those weird bitmasks:

https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events#determining_button_states